lib/memcached/memcached.rb

```diff
@@ -1,736 +0,0 @@
-=begin rdoc
```

Comment:
I havent deleted this yet, because I was copying the interface. same for the rails.rb file